Remotion/Data/DomainObjects.UnitTests/ReflectionUtilityTest.cs

```diff
@@ -355,9 +375,9 @@ public void IsDomainObject_DerivedFromDomainObject_ReturnsTrue ()
     }
 
     [Test]
-    public void IsDomainObject_IDomainObject_ReturnsFalse ()
+    public void IsDomainObject_IDomainObject_ReturnsTrue ()
```

Comment:
Ah, yes, this is new. please make a breaking change ticket for this. (yes, there will be more tickets, but most of those will be signature changes that are easier to discover during a code-scan than a semantic change like this. => we need to document this right away)